### PR TITLE
Add upload attachment mechanism from frontend

### PR DESCRIPTION
**Pull request by XiaoXuxxxx**

### CODE REVIEW

**Comment by XiaoXuxxxx** [comment]
LBTM

**Comment by XiaoXuxxxx** [comment]
We can't go back :)

File: apps/registration/src/components/ImageInputBox.tsx [hunk 1]
```diff
@@ -1,18 +1,27 @@
+import { fetch } from '@/utils/Fetch';
 import { setFormObject } from '@/utils/FormUtil';
 import { StateUpdater, useRef, useState } from "preact/hooks";
 
 const MAX_DISPLAY_FILE_SIZE = 10;
 const MAX_FILE_SIZE_MIB = 1_048_576 * MAX_DISPLAY_FILE_SIZE;
 
+const uploadAttachment = (attachment: File, type: number) => {
```
Comment (riflowth) [inline apps/registration/src/components/ImageInputBox.tsx:8]: Maybe we need to show image URL after upload

**Review by riflowth — APPROVED** [review]
LGTM

**Pull request merged**